src/honey/sql.cljc

```diff
@@ -1877,6 +1877,19 @@
         (into paramsx)
         (into params))))
 
+(defn- between-fn
+  "For both :between and :not-between"
+  [k [x a b]]
+  (let [[sql-x & params-x] (format-expr x {:nested true})
+        [sql-a & params-a] (format-expr a {:nested true})
+        [sql-b & params-b] (format-expr b {:nested true})
+        add-not?     (str/starts-with? (name k) "not")
```

Comment:
Instead of the conditional stuff here, you can use `(sql-kw k)` which will turn `:between` into `BETWEEN` and `:not-between` into `NOT BETWEEN`.

Reply:
Will do!